tests/contrib/gp/test_likelihoods.py

```diff
@@ -17,6 +17,9 @@
 y_binary1D = torch.tensor([0.0, 1.0, 0.0])
 y_binary2D = torch.tensor([[0.0, 1.0, 1.0], [1.0, 0.0, 1.0]])
 binary_likelihood = Binary()
+y_count1D = torch.tensor([0.0, 1.0, 4.0])
+y_count2D = torch.tensor([[5.0, 9.0, 3.0], [4.0, 0.0, 1.0]])
```

Comment:
I have thought that these should be LongTensor, but after checking pytorch's implementation, I agree that we should make them float.